### PR TITLE
Added editable text component for easier editing of address labels

### PR DESCRIPTION
**Pull request by FrederikBolding**
- Added `EditableText` component
- Used `EditableText` to allow for editing of address labels and notes

- [x] Settings page
- [x] Dashboard page
- [x] Use correct icon

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch3523]

**Comment by FrederikBolding** [comment]
Fixed conflicts, ready for review 😄

**Comment by FrederikBolding** [comment]
Yeah I think it works great! What's a reasonable length limit do you think?

**Comment by vilsbole** [comment]
> Thanks, I think it works great! What's a reasonable length limit do you think?

I suggest we arbitrarily choose to ellipse at:
- (28chars - ellipse size) for mobile
- (18chars - ellipse size) for desktop

Which correspond to the available width before the UI starts to wrap or change ratios

**Comment by vilsbole** [comment]
@FrederikBolding 
We got new designs for this!
https://www.figma.com/proto/BY0SWc75teEUZzws8JdgLMpy/MyCrypto-GAU-Master?node-id=5265%3A99923&viewport=325%2C157%2C0.125&scaling=min-zoom

https://mycryptohq.slack.com/archives/G95Q38V9D/p1579281742008000

File: common/v2/components/EditableText.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  height: 100%;
+  align-items: center;
+`;
+
+const EditIcon = styled.img`
+  margin-left: 3px;
+`;
+
+const SInputField = styled.input`
+  border: 1px solid #e8eaed;
```
Comment (vilsbole) [inline common/v2/components/EditableText.tsx]: Let's select or add from COLORS

File: common/v2/components/InputField.tsx [hunk 2]
```diff
@@ -105,17 +119,32 @@ const CustomIconWrapper = styled.div`
   top: 0;
 `;
 
+const CustomIcon = styled.span`
+  display: flex;
+  border-left: 1px solid #d6dce5;
```
Comment (vilsbole) [inline common/v2/components/InputField.tsx]: let's get the hex value from COLORS

**Pull request merged**